### DISCUSSION TITLE
Current File and After Specified Line

### DISCUSSION THREAD

**Discussion by vJourneyman** [chhoumann/quickadd#318] [General]
Is there a way to “Capture to active file” AND Insert after specified line? If I turn on the dialogue for the former, the latter goes away. If I try adding the specified line, then turn on the “to active file” switch, I get an error message saying it can’t find the line (which I copy/pasted from the current file).

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

This combination has worked since v0.18.0: with _Capture to active file_ enabled, the **Write position** dropdown offers **After line…** like any other target. Paste the line there, or enable **Choose heading when capturing** to pick a heading from the note at capture time instead of hard-coding it. There is also _Create line if not found_ if the target line might not exist yet, and **Before line…** as a sibling option.

Docs: https://quickadd.obsidian.guide/docs/Choices/CaptureChoice#insert-after

Closing as resolved.
